include/rangers-api/Application/UI/AppUITextInterface.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace app::ui {
+    class AppUITextInterface : public hh::ui::UITextInterface {
+        text::TextManager* textManager;
```

Comment:
Variable namings should be done in title case.
Pointer types should be identified by a p before the variable name.